### PR TITLE
Dimmer Restore previous value

### PR DESCRIPTION
**Pull request by marcelrv**
addresses #267

Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: ESH-INF/thing/_channels.xml [hunk 1]
```diff
@@ -996,7 +996,18 @@
         <description>The brightness channel allows to control the brightness of a light.
             It is also possible to switch the light on and off.
         </description>
-        <category>DimmableLight</category>
+        <category>DimmableLight</category>`
```
Comment (IOOOTAlan) [inline ESH-INF/thing/_channels.xml]: Is the backtick here (`) a typo?

File: ESH-INF/thing/_channels.xml [hunk 2]
```diff
@@ -996,7 +996,18 @@
...
+         <config-description>
+            <parameter name="config_restoreLastValue" type="boolean">>
```
Comment (IOOOTAlan) [inline ESH-INF/thing/_channels.xml]: There seems to be an extra `>` here at the end of the line.

**Comment by cdjackson** [comment]
Well spotted - strangely the Eclipse editor didn’t seem to highlight these...

**Comment by marcelrv** [comment]
strange indeed.. I've created PR to fix it

**Comment by cdjackson** [comment]
Too slow ;) I fixed it already...

**Pull request merged**